src/lib/commands/birthday.ts

```diff
@@ -1,233 +1,50 @@
-					},
-					{
-						type: ApplicationCommandOptionType.Integer,
-						name: 'day',
-						description: 'Day of birthday',
-						required: true,
-						minValue: 1,
-						maxValue: 31,
-					},
-					{
-						type: ApplicationCommandOptionType.String,
-						name: 'month',
-						description: 'Month of birthday',
-						choices: [
-							{
-								name: 'January',
-								value: '01',
-							},
-							{
-								name: 'February',
-								value: '02',
-							},
-							{
-								name: 'March',
-								value: '03',
-							},
-							{
-								name: 'April',
-								value: '04',
-							},
-							{
-								name: 'May',
-								value: '05',
-							},
-							{
-								name: 'June',
-								value: '06',
-							},
-							{
-								name: 'July',
-								value: '07',
-							},
-							{
-								name: 'August',
-								value: '08',
-							},
-							{
-								name: 'September',
-								value: '09',
-							},
-							{
-								name: 'October',
-								value: '10',
-							},
-							{
-								name: 'November',
-								value: '11',
-							},
-							{
-								name: 'December',
-								value: '12',
-							},
-						],
-						required: true,
-					},
-					{
-						type: ApplicationCommandOptionType.Integer,
-						name: 'year',
-						description: 'Year of birthday',
-						minValue: 1950,
-						maxValue: 2025,
-					},
-				],
-			},
-			{
-				type: ApplicationCommandOptionType.Subcommand,
-				name: 'list',
-				description: 'List all Birthdays in this Discord server',
-			},
-			{
-				type: ApplicationCommandOptionType.Subcommand,
-				name: 'remove',
-				description: 'Remove a birthday - MANAGER ONLY',
-				options: [
-					{
-						type: ApplicationCommandOptionType.User,
-						name: 'user',
-						description: 'The user you want to remove the birthday from',
-						required: true,
-					},
-				],
-			},
-			{
-				type: ApplicationCommandOptionType.Subcommand,
-				name: 'show',
-				description: 'Show the Birthday of you or a other person',
-				options: [
-					{
-						type: ApplicationCommandOptionType.User,
-						name: 'user',
-						description: 'Show the birthday of a specific User',
-					},
-				],
-			},
-			{
-				type: ApplicationCommandOptionType.Subcommand,
-				name: 'test',
-				description: 'Test your current birthday configurations',
-			},
-		],
-	};
-}
+export const monthChoices = [
+	{
+		name: 'January | 1',
+		value: '01',
+	},
+	{
+		name: 'February | 2',
+		value: '02',
+	},
+	{
+		name: 'March | 3',
+		value: '03',
+	},
+	{
+		name: 'April | 4',
+		value: '04',
+	},
+	{
+		name: 'May | 5',
+		value: '05',
+	},
+	{
+		name: 'June | 6',
+		value: '06',
+	},
+	{
+		name: 'July | 7',
+		value: '07',
+	},
+	{
+		name: 'August | 8',
+		value: '08',
+	},
+	{
+		name: 'September | 9',
+		value: '09',
+	},
+	{
+		name: 'October | 10',
+		value: '10',
+	},
+	{
+		name: 'November | 11',
+		value: '11',
+	},
+	{
+		name: 'December | 12',
+		value: '12',
+	},
+] as const;

```

Comment:
why this `as const`?

Reply:
Because this constant must not be changed